src/NHibernate/Impl/StatelessSessionImpl.cs

```diff
@@ -366,7 +358,7 @@ public override FlushMode FlushMode
 
 		public override string BestGuessEntityName(object entity)
 		{
-			using (new SessionIdLoggingContext(SessionId))
+			using (BeginProcess())
```

Comment:
Switch to `BeginProcess` for matching the method on `SessionImpl` side. Cannot be a breaking change because it is calling `GuessEntityName` which, in the case of the stateless session, was doing a status check for a mysterious reason.